Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -40,32 +41,38 @@ type Props = $ReadOnly<{|
   ...IOSProps,
 
   /**
-   * Whether to show the indicator (true, the default) or hide it (false).
+   * Whether to show the indicator (`true`) or hide it (`false`).
    *
-   * See https://reactnative.dev/docs/activityindicator.html#animating
+   * @see https://reactnative.dev/docs/activityindicator.html#animating
+   * @default true
    */
   animating?: ?boolean,
 
   /**
-   * The foreground color of the spinner (default is gray).
+   * The foreground color of the spinner.
    *
-   * See https://reactnative.dev/docs/activityindicator.html#color
+   * @see https://reactnative.dev/docs/activityindicator.html#color
+   * @default {@platform ios} '#999999'
+   * @default {@platform android} null
    */
   color?: ?ColorValue,
 
   /**
-   * Size of the indicator (default is 'small').
-   * Passing a number to the size prop is only supported on Android.
+   * Size of the indicator.
    *
-   * See https://reactnative.dev/docs/activityindicator.html#size
+   * @see https://reactnative.dev/docs/activityindicator.html#size
+   * @default small
+   * @prop {@platform android} number
+   * @prop {@platform ios} string
    */
   size?: ?IndicatorSize,
 |}>;
 
 /**
  * Displays a circular loading indicator.
  *
- * See https://reactnative.dev/docs/activityindicator.html
+ * @see https://reactnative.dev/docs/activityindicator.html
+ * @inheritsProps View {@see https://reactnative.dev/docs/view.html#props}
```

Comment:
I like the idea of `inheritsProps` being its own meta element, since it needs a special rendering case to render it at the top of the props lists.

Nitpicking, but wondering if this should be a` {@see}`. IMO It could just be a markdown link, e.g.

```md
@inherits [View props](https://reactnative.dev/docs/view.html#props)
```

Further, I'd personally name it `@inherits` instead of `@inheritsProps` since we're talking conceptual inheritance that's not specific to React's props, but that's just my aesthetics

Reply:
Yes, I initially thought of using markdown but waiting for the feedback.
 We can go with Markdown syntax.

Reply:
Let's keep the semantics of the `@see` annotations simple as described [here](https://github.com/MLH-Fellowship/react-native/pull/27#discussion_r440148761) and not overload it with additional meaning, that would make parsing unnecessarily complex.

Markdown link is the way to go here :+1: